README.md

```diff
@@ -47,13 +47,27 @@ You can also set `-Zmiri-start-fn` to make Miri start evaluation with the
 
 ## Running Miri on your own project('s test suite)
 
-Install Miri as a cargo subcommand with `cargo install --all-features`, and install
-a full libstd as described above.
+Install Miri as a cargo subcommand with `cargo install --all-features --path .`.
 
-Then, inside your own project, use `MIRI_SYSROOT=~/.xargo/HOST cargo +nightly
-miri` to run your project, if it is a bin project, or run
-`MIRI_SYSROOT=~/.xargo/HOST cargo +nightly miri test` to run all tests in your
-project through Miri.
+Compile your project and its dependencies against a MIR-enabled libstd as described
+above. The easiest way to do this is to copy `xargo/build.sh` and `xargo/Xargo.toml`
+into your own project's root, and to re-run `build.sh`. This ensures that `libstd`
+is compiled with the correct options and that all of your dependencies are built
+against it.
+
+Then use `MIRI_SYSROOT=~/.xargo/HOST cargo +nightly miri` to run your project, if
+it is a bin project, or run `MIRI_SYSROOT=~/.xargo/HOST cargo +nightly miri test`
+to run all tests in your project through Miri.
+
+## Using Rustup To Specify a Specific Nightly
+
+To target a specific nightly, modify the above instructions as follows.
+
+1. Install Miri using `rustup run nightly-2018-10-15 cargo install --all-features --path .`,
+with the date replaced as appropriate.
```

Comment:
A shorter version is `cargo +nightly-2018-10-15 install --all-features --path .`

Reply:
Sure, I'll change this.